en/components/general/update-guide.md

```diff
@@ -51,6 +51,35 @@ Unfortunately not all changes can be automatically updated. Changes below are sp
 
 For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from the "From 6.x .." section apply those changes and work your way up:
 
+## From 14.1.x to 14.2.x
+### General
+- **Breaking Change** - All **igx** prefixes are replaced with **ig** prefixes. For example, **igx-**typography is now **ig-**typography.
```

Comment:
We should clarify that this change only applies to global CSS variables for palette, elevations, typography, etc. As well as the igx-typography class. All component related variables retain the igx prefix.